src/myrtlespeech/builders/task_config.py

```diff
@@ -144,3 +149,65 @@ def target_transform(target):
         train_loader,
         eval_loader,
     )
+
+
+def _create_lr_scheduler(
+    task_config: task_config_pb2.TaskConfig, optimizer: torch.optim
+) -> torch.optim.lr_scheduler:
+    """Builds a ``learning rate scheduler`` and returns it.
+
+    Args:
+        task_config: A :py:class:`task_config_pb2.TaskConfig` protobuf object
+            containing the config for the desired task.
+
+        optimizer: A :py:class:`torch.optim` that is used in the learning rate
+            scheduler initialization
+
+    Returns:
+        A :py:class:`torch.optim.lr_scheduler` instance.
+
+    Raises:
+        :py:class:`ValueError`: On invalid configuration.
+        """
+    lr_scheduler_str = task_config.train_config.WhichOneof(
+        "supported_lr_scheduler"
+    )
+
+    lr_scheduler = None
+    if lr_scheduler_str == "constant_lr":
+        lr_scheduler = torch.optim.lr_scheduler.LambdaLR(
+            optimizer=optimizer, lr_lambda=lambda epoch: 1.0
+        )
```

Comment:
`lr_scheduler = None`? If not, why does this not work?

Reply:
I'm sorry for not having understood what you meant the first time you added this comment. Since the first time you highlighted also other lines, I thought you wanted to have an initialization of the `lr_scheduler` before the list of if-elif-else statements (which I then added). 
Anyway, I have now deleted the LambdaLR and substituted it with a None scheduler in a new commit